tests/puppeteer/test/utils/helpers.js

```diff
@@ -0,0 +1,31 @@
+require('./globals');
+const chai = require('chai');
+chai.use(require('chai-string'));
+global.expect = chai.expect;
+
+const puppeteer = require('puppeteer');
+
+global.test = (name, instructions) => it(name, () => {
+  return instructions().catch();
+});
+
+global.scenario = (name, tests, close = false) =>
+  describe(name, async () => {
+    before(async () => {
+      global.browser = await puppeteer.launch({
+        headless: global.HEADLESS,
+        //slowMo: 100,
```

Comment:
Can you delete slowMo comment here please ?

Reply:
Done !